projects/js-packages/webpack-config/changelog/fix-vs-code-ts-fixes

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fixed
+
+Fixed webpack resolution for internal js-packages by using resolve.alias
```

Comment:
This change does not seem to be present anymore.

Reply:
Removed, thanks.